tests/test_parser.py

```diff
@@ -694,3 +694,41 @@ def test_expand_concatenated_variables(self, dfparser):
             LABEL component="$NAME$VER"
         """)
         assert dfparser.labels['component'] == 'name1'
+
+    def test_multiline(self, dfparser):
+        dfparser.content = dedent("""
+            LABEL a=b c d e \
+                  b="parsing is so hard" \
+                  c=y e s
+        """)
```

Comment:
This test case is not correct. Docker will not parse it. All labels in a statement have to be in the `k=v` form, or all of them have to be in `k v` form. You aren't allowed to mix and match within a single LABEL statement.
```
Sending build context to Docker daemon 2.048 kB
Error response from daemon: Syntax error - can't find = in "c". Must be of the form: name=value
```